dataikuapi/dss/managedfolder.py

```diff
@@ -108,6 +109,21 @@ def put_file(self, path, f):
                 "POST", "/projects/%s/managedfolders/%s/contents/%s" % (self.project_key, self.odb_id, utils.quote(path)),
                 "", f)
 
+    def upload_folder(self, path, folder):
+        """
+        Upload folder and its content as path in the managed folder.
+
+        :param str folder: path to the folder (absolute or relative)
+        :param str path: the path of the folder in the managed folder
```

Comment:
[Nitpick] Keep the argument order to limit confusion when reading this doc within the code. And can use the opportunity to clarify with "source" and "destination" wording.

```suggestion
        :param str path: the destination path of the folder in the managed folder
        :param str folder: path  (absolute or relative) of the source folder to upload
```